internal/activities/validate_file_formats.go

```diff
@@ -73,19 +76,55 @@ func (a *ValidateFileFormats) Execute(
 		if err != nil {
 			return err
 		}
+
 		if d.IsDir() {
 			return nil
 		}
+
 		ff, err := sf.Identify(p)
 		if err != nil {
 			return fmt.Errorf("identify format: %v", err)
 		}
+
+		// Determine PREMIS event summary detail/outcome and note failure.
+		detail := ""
+		outcome := "valid"
+
 		if _, exists := allowed[ff.ID]; !exists {
+			detail = fmt.Sprintf("file format %q not allowed: %q", ff.ID, p)
+			outcome = "invalid"
+
 			failures = append(
```

Comment:
This is short enough to be a one-liner now:
```go
failures = append(failures, detail)
```

Reply:
Fixed!